.github/workflows/dev-deploy-inbound-mail.yml

```diff
@@ -24,9 +24,7 @@ jobs:
         name: ['novu/inbound-mail-ee', 'novu/inbound-mail']
     uses: ./.github/workflows/reusable-inbound-mail-e2e.yml
     with:
-      ee: ${{ contains (matrix.name,'ee') }}
-      submodules: ${{ contains (matrix.name,'ee') }}
-      submodule_branch: "next"
```

Comment:
IMO one flash that says that we need to run ee is enough if not we could use the matrix name.